client/components/Categories/component.js

```diff
@@ -56,10 +70,13 @@ export default class CategoriesComponent extends Component {
         <form className="form" onSubmit={this.handleSubmit}>
```

Comment:
I think that since we have 2 forms we should give them more descriptive names instead of adding 2.